### PR TITLE
feat(search): #1674 show references to datasets on concept details page

### PR DESCRIPTION
**Pull request by azaxarov**

### CODE REVIEW

File: applications/search/src/api/concepts.js [hunk 1]
```diff
@@ -11,11 +12,26 @@ export const conceptsSearchUrl = query =>
     { addQueryPrefix: true }
   )}`;
 
-export const getConcept = id =>
-  axios
+async function getDatasetsForConcept(conceptId) {
```
Comment (markkopaas) [inline applications/search/src/api/concepts.js]: litt enklere å forstå hvis vi bruker enkle datastrukturer. eksport her.

File: applications/search/src/api/concepts.js [hunk 2]
```diff
@@ -11,11 +12,26 @@ export const conceptsSearchUrl = query =>
...
+    .then(r => r.map(hit => hit._source))
+    .catch(() => []);
+}
+
+export const getConcept = async id => ({
+  ...(await axios
     .get(`${conceptsUrlBase}/${id}`)
     .then(response => response.data)
-    .catch(e => console.error(JSON.stringify(e)));
+    .catch(e => console.error(JSON.stringify(e)))),
```
Comment (markkopaas) [inline applications/search/src/api/concepts.js]: .catch(console.error)

**Review by markkopaas — CHANGES_REQUESTED** [review]

File: applications/search/src/api/concepts.js (hunk 1 above)
Comment (azaxarov) [reply applications/search/src/api/concepts.js]: Done

File: applications/search/src/api/concepts.js (hunk 2 above)
Comment (azaxarov) [reply applications/search/src/api/concepts.js]: Done

**Review by markkopaas — APPROVED** [review]

**Pull request merged**